py-polars/tests/unit/test_datelike.py

```diff
@@ -2600,7 +2626,6 @@ def test_infer_iso8601(iso8601_format: str) -> None:
         .replace("%3f", "123")
         .replace("%6f", "123456")
         .replace("%9f", "123456789")
-        .replace("%Z", "Z")
```

Comment:
not strictly related to this PR, but the `infer_pattern_datetime_single` doesn't handle timezone-aware (yet?), so this line wasn't  necessary